Lib/test/test_zipfile.py

```diff
@@ -2442,6 +2491,18 @@ def zipfile_abcde(self):
         with self.subTest():
             yield add_dirs(build_abcde_files())
 
+    def zipfile_abcdef(self):
```

Comment:
These tests should describe what it is they're attempting to test (or have a docstring to clarify).